src/ansible_navigator/actions/inventory.py

```diff
@@ -142,10 +142,8 @@ def _set_inventories_mtime(self) -> None:
             if os.path.isdir(inventory):
                 mtimes.append(
                     max(
-                        (
-                            os.path.getmtime(e)
-                            for e in glob.glob(os.path.join(inventory, "**"), recursive=True)
-                        )
+                        os.path.getmtime(e)
```

Comment:
As a possible (not really in the scope, though) improvement — use longer variable names. Even in comprehensions/generator expressions: just looking at this patch, I can't immediately guess what's `e` here, for example. It would be nice to see a more meaningful name.

Reply:
(resolving so that it doesn't block gating)